tslint.json

```diff
@@ -8,18 +7,12 @@
 		// Better to have functionally similar items be close together.
 		"member-ordering": false,
 		"no-class": false,
-		"no-delete": true,
 		"no-expression-statement": false,
 		"no-if-statement": false,
-		"no-let": true,
-		"no-loop-statement": true,
 		"no-method-signature": false,
-		"no-mixed-interface": true,
 		"no-object-mutation": false,
-		"no-parameter-reassignment": true,
```

Comment:
`no-parameter-reassignment` and `no-var-keyword` are worth keeping imo. are we planning to re-introduce later?

Reply:
No, we are actually keeping them. Just, they are enabled by default, so there is no need those statements to be in tslint.json.